hudi-client/hudi-client-common/src/test/java/org/apache/hudi/table/upgrade/TestSevenToEightUpgradeHandler.java

```diff
@@ -122,4 +116,48 @@ void testPropertyUpgrade() {
     assertTrue(tablePropsToAdd.containsKey(KEY_GENERATOR_CLASS_NAME));
     assertTrue(tablePropsToAdd.containsKey(KEY_GENERATOR_TYPE));
   }
+
+  @ParameterizedTest
+  @CsvSource({
+      "com.example.CustomPayload, , CUSTOM, " + HoodieRecordMerger.PAYLOAD_BASED_MERGE_STRATEGY_UUID + ", com.example.CustomPayload",
+      "com.example.CustomPayload, preCombineFieldValue, CUSTOM, " + HoodieRecordMerger.PAYLOAD_BASED_MERGE_STRATEGY_UUID + ", com.example.CustomPayload",
+      "org.apache.hudi.metadata.HoodieMetadataPayload, , CUSTOM, " + HoodieRecordMerger.PAYLOAD_BASED_MERGE_STRATEGY_UUID + ", org.apache.hudi.metadata.HoodieMetadataPayload",
+      "org.apache.hudi.metadata.HoodieMetadataPayload, preCombineFieldValue, CUSTOM, " + HoodieRecordMerger.PAYLOAD_BASED_MERGE_STRATEGY_UUID + ", org.apache.hudi.metadata.HoodieMetadataPayload",
+      "org.apache.hudi.common.model.OverwriteWithLatestAvroPayload, , COMMIT_TIME_ORDERING, " + HoodieRecordMerger.COMMIT_TIME_BASED_MERGE_STRATEGY_UUID
+          + ", org.apache.hudi.common.model.OverwriteWithLatestAvroPayload",
+      "org.apache.hudi.common.model.DefaultHoodieRecordPayload, , EVENT_TIME_ORDERING, " + HoodieRecordMerger.EVENT_TIME_BASED_MERGE_STRATEGY_UUID
```

Comment:
Could we hardcode the UUID?  The variable name has changed from 0.14 to 1.0 release so it's better to use hardcoded string as parameters and add javadocs on the mapping.